src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -569,12 +569,17 @@ def get_missing_image(self):
         # field.
         return MissingProductImage()
 
+    def all_images(self):
```

Comment:
Suggest renaming this to `get_all_images()` to be consistent with the other methods that are defined above. Otherwise looks good.